### DISCUSSION TITLE
[tabs] vertical variant

### DISCUSSION THREAD

**Discussion by mauritskorse** [livewire/flux#257] [Showcase]
Vertical tabs with tabpanel on the right (or left side).
![image](https://github.com/user-attachments/assets/c34774cd-7517-4444-90aa-8504aeb8383f)

Currently I have done it like this. 
```
<flux:tab.group class="flex flex-row gap-8">
    <flux:tabs class="flex-col !border-b-0 pr-4 border-r">
        <flux:tab name="profile" icon="user" class="!border-b-0 border-l-[2px]">Profile</flux:tab>
        <flux:tab name="settings" icon="cog-6-tooth" class="!border-b-0 border-l-[2px]">Settings</flux:tab>
        <flux:tab name="security" icon="shield-check" class="!border-b-0 border-l-[2px]">Security</flux:tab>
        <flux:tab name="billing" icon="banknotes" class="!border-b-0 border-l-[2px]">Billing</flux:tab>
    </flux:tabs>

  <flux:tab.panel name="profile" class="items-start !pt-0 space-y-4">
    ...
  </flux:tab.panel>
```

**Comment by benjibee** [comment]
@joshhanley some good feature requests getting lost here!

**Reply by joshhanley** [reply]
@benjibee what do you mean they are getting lost?

**Reply by benjibee** [reply]
I was clicking through the different pages of discussions and seeing some interesting feature requests without much interaction. Like this one from Oct. 2nd. Maybe "getting lost" is a bit dramatic, just don't know what's on the team's radar!

**Reply by joshhanley** [reply]
@benjibee ah right, yeah we see them all but yeah can be tough to keep track because there are so many 🙂

**Comment by melokullc** [comment]
Thank you for sharing this tabs variant, I was just searching for this for a project!
